In [1]:
import numpy as np
import pandas as pd

In [2]:
df_acuicultura = pd.read_excel("https://www.subpesca.cl/portal/619/articles-92935_listado.xlsx")

In [3]:
#Tamaño dataset concesiones acuicultura
df_acuicultura.shape

(3233, 19)

In [4]:
#Muestra las cinco primeras filas del dataset
df_acuicultura.head()

codigoCentro    numPert                          Comuna  \
0         90201  200091005  CARAHUE                          
1        104477  219101021  MAULLIN                          
2        104476  207105039  ANCUD                            
3        104475  206101181  MAULLIN                          
4        104473  209103041  QUEMCHI                          

                   Region                                          Titulares  \
0  REGIÓN DE LA ARAUCANÍA         5692719-0: AVILA GALDAMES, HECTOR FERNANDO   
1     REGIÓN DE LOS LAGOS          18844230-7: VARGAS MANSILLA MARIA IGNACIA   
2     REGIÓN DE LOS LAGOS         71780900-9: SINDICATO TRABAJADORES CHAULAR   
3     REGIÓN DE LOS LAGOS  10512152-0: CARDENAS BUSTAMANTE, ROBINSON EFIL...   
4     REGIÓN DE LOS LAGOS  10121206-8: PEÑA IMILMAQUI, JUAN VICTOR, 11595...   

   NumeroResSSP FechaResSSP ResultadoResolucionSSP  NumeroResSSFFAA  \
0          1122  2023-05-08                Aprueba              441   
1          2111  2022-10-18                Aprueba               39   
2           645  2023-03-13                Aprueba             2723   
3           896  2023-04-04                Aprueba             2726   
4          1418  2023-06-23                Aprueba             2727   

  FechaResSSFFAA ResultadoResolucionSSFFAA  \
0     2024-03-26                   Aprueba   
1     2024-01-11                   Aprueba   
2     2023-12-01                   Aprueba   
3     2023-12-01                   Aprueba   
4     2023-12-01                   Aprueba   

                                          EspeciesPT     GrupoPT  \
0                             CHOLGA, CHORITO, CHORO   MITILIDOS   
1  CAROLA, CHASCA, CHASCON O HUIRO NEGRO, CHICORE...  MACROALGAS   
2  CAROLA, CHASCA, CHASCON O HUIRO NEGRO, CHICORE...  MACROALGAS   
3  CAROLA, CHASCA, CHASCON O HUIRO NEGRO, CHICORE...  MACROALGAS   
4                             CHOLGA, CHORITO, CHORO   MITILIDOS   

  GRUPO ESPECIE AGRUPADO                                          toponimio  \
0               MOLUSCOS    (1): RÍO IMPERIAL, AL OESTE DE SECTOR SAN PABLO   
1                  ALGAS          (1): RIO MAULLIN RIBERA NORTE, PARCELA R4   
2                  ALGAS  (1): ESTERO CHAULAR, AL OESTE DE PUNTA LARGA. ...   
3                  ALGAS                   (1): RÍO QUENUIR, SECTOR QUENUIR   
4               MOLUSCOS  (1): ESTERO BUTATIQUE, AL ESTE DE ISLA CHEMLLI...   

  tipoBarrio barrio                                  TipoPorcion  \
0                                              (1): AGUA Y FONDO   
1                                              (1): AGUA Y FONDO   
2                    (1): PLAYA , (2): PLAYA , (3): AGUA Y FONDO   
3                                              (1): AGUA Y FONDO   
4                                              (1): AGUA Y FONDO   

  superficieTotal  
0             1.5  
1             0.3  
2               4  
3            7.39  
4            2.97

In [5]:
df_acuicultura.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 3233 entries, 0 to 3232
Data columns (total 19 columns):
 #   Column                     Non-Null Count  Dtype         
---  ------                     --------------  -----         
 0   codigoCentro               3233 non-null   int64         
 1   numPert                    3233 non-null   object        
 2   Comuna                     3233 non-null   object        
 3   Region                     3233 non-null   object        
 4   Titulares                  3233 non-null   object        
 5   NumeroResSSP               3233 non-null   int64         
 6   FechaResSSP                3233 non-null   datetime64[ns]
 7   ResultadoResolucionSSP     3233 non-null   object        
 8   NumeroResSSFFAA            3233 non-null   int64         
 9   FechaResSSFFAA             3233 non-null   datetime64[ns]
 10  ResultadoResolucionSSFFAA  3233 non-null   object        
 11  EspeciesPT                 3233 non-null   object        
 12  GrupoP

El dataset de concesiones de acuicultura vigentes a junio 2024 contiene 3233 filas y 19 columnas. Las columnas son:

**codigoCentro**, cuyo tipo de dato es integer y representa al código asignado a cada centro de cultivo de especies acuícolas de acuerdo al Registro Nacional de Acuicultura (RNA).

**numPert**, cuyo tipo de dato es objeto (string) y representa al Número de Pertenencia original de un proyecto de concesión acuícola (código de la solicitud de inscripción de la concesión).

**Comuna**, cuyo tipo de dato es objeto (string) y representa a la comuna en la cual está ubicado un centro de cultivo de especies acuícolas.

**Region**, cuyo tipo de dato es objeto (string), y representa a la región en la cual se ubica un centro de cultivo acuícola.

**Titulares**, cuyo tipo de dato es objeto (string), y representa al RUT y nombre de cada titular (persona natural, empresa o universidad) de una concesión de acuicultura.

**NumeroResSSP**, cuyo tipo de dato es objeto (string), y representa al número de la resolución de la Subsecretaría de Pesca que autoriza o rechaza una concesión de acuicultura.

**FechaResSSP**, cuyo tipo de dato es datetime (fecha), y representa a la fecha de una resolución de la Subsecretaría de Pesca que autoriza o rechaza una concesión de acuicultura.

**ResultadoResolucionSSP**, cuyo tipo de dato es objeto (string), e indica si la Subsecretaría de Pesca autoriza o rechaza una concesión de acuicultura.

**NumeroResSSFFAA**, cuyo tipo de dato es objeto (string), y representa al número de la resolución de la Subsecretaría para las Fuerzas Armadas que autoriza o rechaza una concesión de acuicultura.

**FechaResSSFFAA**, cuyo tipo de dato es datetime (fecha), y representa a la fecha de una resolución de la Subsecretaría para las Fuerzas Armadas que autoriza o rechaza una concesión de acuicultura.

**ResultadoResolucionSSFFAA**, cuyo tipo de dato es objeto (string), e indica si la Subsecretaría para las Fuerzas Armadas autoriza o rechaza una concesión de acuicultura.

**EspeciesPT**, cuyo tipo de dato es objeto (string), y representa a o las especies mencionadas en el Proyecto Técnico requerido para obtener una concesión de acuicultura.

**GrupoPT**, cuyo tipo de dato es objeto (string), y representa al grupo específico autorizado al que pertenecen las especies mencionadas en el Proyecto Técnico requerido para obtener una concesión de acuicultura (por ej. si se trata de un tipo específico de pez tales como los salmónidos, alga del tipo macroalga o molusco del grupo de los choritos).

**GRUPO ESPECIE AGRUPADO**, cuyo tipo de dato es objeto (string), y representa al grupo genérico (peces, algas o moluscos) al que pertenecen las especies mencionadas en el Proyecto Técnico requerido para obtener una concesión de acuicultura.

**toponimio**, cuyo tipo de dato es objeto (string), y representa al lugar geográfico específico (por ej. río, estero, isla, canal, bahía) en el cual se desarrolla una concesión de acuicultura.

**tipoBarrio**,  cuyo tipo de dato es objeto (string), y representa al tipo de agrupación de concesión (por ej. de salmones o moluscos).

**barrio**, cuyo tipo de dato es objeto (string), y consiste en la identificación del número de barrio o agrupación de concesiones de acuicultura (ACS).

**TipoPorcion**,  cuyo tipo de dato es objeto (string), y representa al nombre del tipo de concesión (playa, terreno de playa, agua y fondo).

**superficieTotal**,  cuyo tipo de dato es objeto (string), y representa a la superficie total en hectáreas de los polígonos (áreas de mar donde se ubica la concesión) de acuicultura soliciticados o propuesta de solicitud en concesión.